service-front/assets/js/opg/session-timeout-dialog.js

```diff
@@ -108,32 +132,48 @@ var SessionTimeoutDialog = function (options) {
 
         // Keep the session alive
         $.get(this.keepSessionAliveUrl)
-            .complete(function () {
-                // restart countdown to get new session expiry in one minute
-                that.checkSessionState();
-            });
+         .complete(function () {
+             // restart countdown
+             that.checkSessionState();
+         });
 
         this.releaseNavigation();
     };
 
-    this.trapNavigation = function () {
-        continueButton.keydown(function (e) {
-            if (e.key === 'Tab' && e.shiftKey) {
-                e.preventDefault();
-                logoutButton.focus();
+    this.trapNavigation = function (focusables) {
+        var currentElementIndex = 0;
+        focusables[currentElementIndex].focus();
+
+        var lastFocusableIndex = focusables.length - 1;
+
+        this.element.keydown(function (e) {
+            // capture only tab events on this dialog
+            if (e.key !== 'Tab' && e.keyCode !== 9) {
+                return true;
             }
-        });
-        logoutButton.keydown(function (e) {
-            if (e.key === 'Tab' && !e.shiftKey) {
-                e.preventDefault();
-                continueButton.focus();
+            e.preventDefault();
+
+            var direction = 1;
+            if (e.shiftKey) {
+                direction = -1;
             }
```

Comment:
```suggestion
            var direction = (e.shiftKey ? -1 : 1); 
```
might be more succinct, though is personal choice...

Reply:
I think I avoided this in case it's not supported by older browsers; but as this file is compiled by the minifier anyway, that's maybe not a consideration.

Reply:
it is probably  minified this way anyway, like I said just a personal preference of mine so no issues 👍🏻